docs/upgrade.html

```diff
@@ -19,6 +19,56 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_3_0" href="#upgrade_3_3_0">Upgrading to 3.3.0 from any version 0.8.x through 3.2.x</a></h4>
+
+<p><b>If you are upgrading from a version prior to 2.1.x, please see the note below about the change to the schema used to store consumer offsets.
```

Comment:
Where is this section coming from? Seems irrelevant to KAFKA-13846?

Reply:
Oh I added this as I couldn't find any section for 3.3. I noticed that all 3+ versions has this upgrade related advice and hence added. Should I remove it?

Reply:
@guozhangwang let me know if i should remove the upgrade section...

Reply:
@guozhangwang , i removed the upgrade section to 3.3 section from the docs. Plz review whenever you get the chance. Thanks!

Reply:
Sorry @vamossagar12 this somehow slipped on my radar, will resume reviewing again.

Reply:
no worries :) Thanks !